lib/utilities/platform-checker.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var semver = require('semver');
+
+var LOWER_RANGE = '0.12.0';
+var UPPER_RANGE = '4.2.1';
```

Comment:
This should be <5.0.0, because we want point releases like 4.2.2 to pass without a code change.